upgrading/topics/keycloak/changes-19_0_0.adoc

```diff
@@ -111,6 +111,21 @@ Contains all REST endpoints that directly operate on the legacy store, and have
 These modules will be available as long as legacy stores will be supported.
 After that period, they will be removed.
 
+This change impacts deployments of existing user storage providers. If your user storage provider is deployed
+as a WAR archive, you need to add the `META-INF/jboss-deployment-structure.xml` file into that archive
+stating the modified dependencies, i.e. with the following contents:
```

Comment:
Latin words and abbreviations like `i.e.` should avoided in the current style guide. 

```suggestion
stating the modified dependencies as shown below:
```